src/main/java/io/github/cottonmc/energy/api/EnergyStorage.java

```diff
@@ -13,8 +13,9 @@
     int getCurrentEnergy();
 
     /**
-     * @return the highest tier of energy this device is capable of working with under any circumstances.
+     * @return any energy types this device is capable of accepting.
+     * The first item in the array is seen as the "primary" energy type.
      */
-    PacketTier getPacketTier();
+    EnergyType[] getEnergyTypes();
```

Comment:
The advantage of having it here is so that a probe can tell what the primary energy type for the machine is and display that in a FruitPhone or WAILA-like.

Reply:
I can sort of see some probe access benefits, but not for multiple energy types; maybe just one getPrimaryEnergyType as a hint if you don't know what to look for.